pytorch_lightning/plugins/apex.py

```diff
@@ -20,12 +20,10 @@
 from pytorch_lightning.core.lightning import LightningModule
 from pytorch_lightning.plugins.precision_plugin import PrecisionPlugin
 from pytorch_lightning.utilities.distributed import rank_zero_warn
-from pytorch_lightning.utilities import AMPType
+from pytorch_lightning.utilities import AMPType, APEX_AVAILABLE
 
-try:
+if APEX_AVAILABLE:
     from apex import amp
-except ImportError:
-    amp = None
```

Comment:
here, `amp` is used below. will it be a problem??

Reply:
here it is fine, as there is no diff if it fails because it is not imported or None has nom attribute so not callable :D